presto-orc/src/main/java/com/facebook/presto/orc/metadata/OrcFileTail.java

```diff
@@ -80,4 +85,23 @@ public int getMetadataSize()
     {
         return metadataSize;
     }
+
+    public Optional<DwrfStripeCacheData> getDwrfStripeCacheData()
+    {
+        return dwrfStripeCacheData;
+    }
+
+    public int getDwrfStripeCacheSize()
```

Comment:
This should be a private method, as it replaces empty with 0 size. If it needs to be a public method, it should return OptionalInt to preserve semantics.

Reply:
Made it private. The original purpose was to use it in the `com.facebook.presto.raptor.storage.StorageModule#createOrcFileTailSource`, but I introduced getTotalSlicesSize instead.